scripts/write-cron-teamcity-backup-sync.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+if [ ! -f scripts/sync-backups.sh ]; then
+    echo "Run this from the montagu-ci directory"
+    exit 1
+fi
+
+PATH_VAGRANT=$PWD
+DEST=/etc/cron.daily/teamcity-backup-sync
+
+cat <<EOF > $DEST
```

Comment:
What does this black magic do?

Reply:
I've added an explanation but I'm not sure what part is the black magic!

Reply:
Ah, given

> I think what you are doing is cating an inline crontab into the VM. But I don't understand what PATH_VAGRANT is doing.

I've added a further explanation about what is being affected here - this is run on the _host_ machine